### ISSUE TITLE
Getting Diff in service port section

### ISSUE THREAD

**Issue by dbadola** [terraform-providers/terraform-provider-fortios#12]
I am using below code

```
resource "fortios_firewall_object_service" "proxy" {
  name          = "tcp8080"
  category      = "General"
  protocol      = "TCP"
  tcp_portrange = "8080"
  comment       = "Proxy"
}

resource "fortios_firewall_object_service" "udp" {
  name          = "udp8080"
  category      = "General"
  protocol      = "UDP"
  udp_portrange = "8080"
  comment       = "Proxy"
}
```

after apply when running the terraform plan again . I am getting below output

```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  ~ fortios_firewall_object_service.proxy
      protocol:        "TCP/UDP/SCTP" => "TCP"

  ~ fortios_firewall_object_service.udp
      protocol:        "TCP/UDP/SCTP" => "UDP"
      protocol_number: "17" => "6"


Plan: 0 to add, 2 to change, 0 to destroy.
```

* Protocol is being set to "TCP/UDP/SCTP"
* For UDP  protocol is being set as 6 instead of 17

**Comment by fwiethof** [comment]
The same problem occurs here as well.

**Comment by frankshen01** [comment]
When you want to set TCP or UDP or SCTP, you need to set the protocol to "TCP/UDP/SCTP", and then the system will automatically identify the specific protocol through tcp_portrange, udp_portrange, sctp_portrange. Please refer to "firewall service custom" section of the FortiOS HandBook CLI Reference.  And the protocol_number problem will be fixed.

**Comment by frankshen01** [comment]
This has been released in version 1.1.0 of the Terraform FortiOS provider.
